schedule/functional/extra_tests_textmode.yaml

```diff
@@ -66,6 +66,7 @@ conditional_schedule:
                 - '{{steamcmd}}'
                 - console/libqca2
                 - console/python_scientific
+                - console/python_flake8
```

Comment:
Does this test require phub or not ?

Reply:
This test requires phub extension on SLE releases, hence adding it to `schedule/functional/extra_tests_textmode_phub.yaml`.
And it is part of tumbleweed releases as default, hence adding it in `schedule/functional/extra_tests_textmode.yaml` under tumbleweed_tests.